src/JestProcessManagement/JestProcess.ts

```diff
@@ -72,13 +72,13 @@ export class JestProcess {
     this.onExitCallback = callback
   }
 
-  public onJestEditorSupportEvent(event, callback) {
+  public onJestEditorSupportEvent(event: string, callback: (...args: any) => void) {
     this.jestSupportEvents.set(event, callback)
     this.runner.on(event, callback)
     return this
   }
 
-  public stop() {
+  public stop(): Promise<undefined> {
```

Comment:
do you mean `Promise<void>`?

Reply:
Ah yes sorry, this was my first foray into Typescript, and I was unsure how to type the existing empty-resolve Promise.